internal/local/structure.go

```diff
@@ -373,13 +401,25 @@ func writeServices(rootDir string, services []ServiceStructure) error {
 				return err
 			}
 		}
+
+		// Rules
 		for _, rule := range svc.Rules {
 			data, err := MarshalJSON(rule)
 			if err != nil {
 				return err
 			}
+
+			// V1 Rules have an exported "name" field
```

Comment:
is this actually true?  I'm doubting myself now but I thought I saw in `importexport` that we never exported the "name" field...